typescript/package.json

```diff
@@ -48,6 +49,8 @@
     "eslint-plugin-jest": "25.2.4",
     "eslint-plugin-prettier": "4.0.0",
     "jest": "27.3.1",
+    "json-diff": "^0.7.1",
+    "json-stringify-pretty-compact": "^3.0.0",
```

Comment:
These shouldn't be added here; there is a separate package.json for the conformance tests. And `lodash` should be added there too because I don't think it's currently listed as an explicit dependency.